numpy/core/src/npysort/radixsort.c.src

```diff
@@ -0,0 +1,237 @@
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#include "npy_sort.h"
+#include "npysort_common.h"
+#include <stdlib.h>
+
+/*
+ *****************************************************************************
+ **                            INTEGER SORTS                                **
+ *****************************************************************************
+ */
+
+
+/**begin repeat
+ *
+ * #TYPE = BOOL, BYTE, UBYTE, SHORT, USHORT, INT, UINT, LONG, ULONG,
+ *         LONGLONG, ULONGLONG#
+ * #suff = bool, byte, ubyte, short, ushort, int, uint, long, ulong,
+ *         longlong, ulonglong#
+ * #type = npy_ubyte, npy_ubyte, npy_ubyte, npy_ushort, npy_ushort, npy_uint,
+ *         npy_uint, npy_ulong, npy_ulong, npy_ulonglong, npy_ulonglong#
+ * #sign = 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0#
+ * #floating = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0#
+ */
+
+// Reference: https://github.com/eloj/radix-sorting#-key-derivation
+#if @sign@
+    #if @floating@
```

Comment:
Can we add a comment on why this (handling floats) is included but not actually enabled?

Reply:
I’m not sure... Tests succeeded on macOS, but not on Windows/Linux, I didn’t attempt to debug too much after that. Is there an expert on bit-level differences in platforms with regard to floating point representations?

Reply:
The issue is, regular tests succeeded... Some of the other tests relying on sort failed. Maybe floating-point normalisation?

Reply:
It's fine that we haven't figured this out yet, but let's add a note to summarize what you just said for future readers/maintainers of this code.

Reply:
Done!